cpp/include/cudf/ast/detail/operators.hpp

```diff
@@ -819,7 +820,17 @@ struct operator_functor<ast_operator::NOT, false> {
 template <typename To>
 struct cast {
   static constexpr auto arity{1};
-  template <typename From>
+  template <typename From, typename cuda::std::enable_if_t<is_fixed_point<From>()>* = nullptr>
```

Comment:
We use `std::enable_if_t` elsewhere in this file. Let's match.
```suggestion
  template <typename From, typename std::enable_if_t<is_fixed_point<From>()>* = nullptr>
```